src/libraries/Microsoft.PowerFx.Core/Functions/Blob/BlobContent.cs

```diff
@@ -8,17 +8,17 @@
 
 namespace Microsoft.PowerFx.Core.Functions
 {
-    internal abstract class BlobContent
+    public abstract class BlobContent
```

Comment:
>public [](http://example.com/codeflow?start=4&length=6)

note you'll need to update PublicSurface_Tests . 

(you'll see this failure when you run tests)